### PR TITLE
Checklist: rename `values` prop to `value`

### PR DESCRIPTION
**Pull request by alexcjohnson**
This is one of the breaking changes planned for the Dash 1.0 release.

Also a couple of little prop type & doc fixes for the `value` prop in `Dropdown` and `RadioItems`.

### CODE REVIEW

File: test/test_integration.py [hunk 1]
```diff
@@ -838,7 +838,7 @@ def test_gallery(self):
                     {'label': 'San Francisco', 'value': 'SF'},
                     {'label': u'北京', 'value': u'北京'}
```
Comment (byronz) [inline test/test_integration.py]: we should leave an eastern egg here with {'label': u'北京', 'value': u'帝都'}

Comment (alexcjohnson) [reply test/test_integration.py]: I liked your original comment - it would be an eastern egg indeed 😉 feel free to add it!

Comment (byronz) [reply test/test_integration.py]: but you might not get the egg? or you level of mandarin is beyond my imagination :)

Comment (alexcjohnson) [reply test/test_integration.py]: My mandarin is only as good as google translate's. But I have no problem with spreading the fun around to more languages - we have plenty of little English jokes scattered around our tests!

**Pull request merged**